### DISCUSSION TITLE
if you are in search of star 👇

### DISCUSSION THREAD

**Discussion by warriorwizard** [Schweinepriester/github-profile-achievements#208] [Q&A] [answered]
please leave a star here  🙏🙏 : https://github.com/warriorwizard/warriorwizard
i will help others in search of star,
btw thanks in advance

~ add a star and add leave your repo link here we I will start it with my 2 github ids

**Comment by number0x0001** [comment]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by warriorwizard** [reply]
@number0x0001 starred bro

**Reply by warriorwizard** [reply]
Done,
Leave a star a star here  🙏🙏 : https://github.com/warriorwizard/warriorwizard
and mention ur repo link down below👇

**Reply by cesarreyes3** [reply]
here please: https://github.com/masbug/flysystem-google-drive-ext

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Please leave a star here: https://github.com/bacca87/great2

**Reply by bensonarafat** [reply]
Pls help me https://github.com/codepymdev/grad

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Reply by elijahpereira** [reply]
done https://github.com/elijahpereira/elijahpereira.github.io

**Comment by CustomEntity** [comment]
https://github.com/CustomEntity/crNormz

I star back

**Reply by nel0x** [reply]
@CustomEntity  Done! Please leave one at mine https://github.com/nel0x/arch-installer

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by warriorwizard** [reply]
Done,
Leave a star a star here  🙏🙏 : https://github.com/warriorwizard/warriorwizard
and mention ur repo link down below👇

**Reply by cesarreyes3** [reply]
Done, https://github.com/masbug/flysystem-google-drive-ext

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by ArielMAJ** [reply]
@CustomEntity I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by Imperfective** [comment]
thx https://github.com/Imperfective/Imperfective

**Reply by warriorwizard** [reply]
Done,
Leave a star a star here  🙏🙏 : https://github.com/warriorwizard/warriorwizard
and mention ur repo link down below👇

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by guexs** [reply]
@imperfective done, give me a star please https://github.com/wannderlust/wannderlust

**Reply by cesarreyes3** [reply]
Done, 
Please https://github.com/masbug/flysystem-google-drive-ext

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by bensonarafat** [reply]
Pls help me  https://github.com/codepymdev/grad

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Comment by warriorwizard** [comment, marked as answer]
please leave a starr here too 🙏🙏 : https://github.com/warriorwizard/warriorwizard
i will surely star others,
btw thanks in advance

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by cesarreyes3** [reply]
Done,
Please: https://github.com/masbug/flysystem-google-drive-ext

**Reply by bensonarafat** [reply]
Pls help me https://github.com/codepymdev/grad

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Comment by noowz** [comment]
https://github.com/Fastxyz/brawlwrap.js

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by cesarreyes3** [reply]
Done, 
Please: https://github.com/masbug/flysystem-google-drive-ext

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Comment by guexs** [comment]
Please: https://github.com/wannderlust/wannderlust

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by erikn69** [reply]
@number0x0001 you didn't give a star to anyone, 
must be reciprocal

**Reply by cesarreyes3** [reply]
Done, 
Please: https://github.com/masbug/flysystem-google-drive-ext

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by vinograd0v** [reply]
https://github.com/mmanosalva/SublimeTeX done

**Comment by erikn69** [comment]
Please help here: https://github.com/erikn69/laravel-google-drive-ext-demo

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by erikn69** [reply]
@number0x0001 you didn't give me one

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Comment by Chameleon-Coder** [comment]
Please leave a star here: https://github.com/Chameleon-Coder/Chameleon-Coder

**Reply by cesarreyes3** [reply]
Done
Please: https://github.com/masbug/flysystem-google-drive-ext

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by bensonarafat** [reply]
Pls help me https://github.com/codepymdev/grad

**Comment by sulincix** [comment]
Please leave a star here: https://github.com/sulincix/debian-subsystem

**Reply by PaulSpoerry** [reply]
Done. Here's mine! https://github.com/PaulSpoerry/test

**Reply by bacca87** [reply]
Done! Please leave a star here: https://github.com/bacca87/great2

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Comment by bacca87** [comment]
@warriorwizard done!
Please leave a star here: https://github.com/bacca87/great2

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Comment by 3kh0** [comment]
@warriorwizard please mark this as a answer I am trying to get Galaxy brain

**Comment by ArielMAJ** [comment]
@warriorwizard I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@warriorwizard I still haven't received your star, so I'll be removing yours. Lmk when you star me so I'll star you back again!

**Reply by ArielMAJ** [reply]
Ty! @mmanosalva I starred you back!

Congratz on your Starstruck badge!

**Comment by bensonarafat** [comment]
Pls help me https://github.com/codepymdev/grad

**Reply by ArielMAJ** [reply]
@bensonarafat I still haven't received your star, so I'll be removing yours. Lmk when you star me so I'll star you back again! https://github.com/ArielMAJ/test.me

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Reply by vinograd0v** [reply]
done https://github.com/mmanosalva/SublimeTeX

**Comment by vinograd0v** [comment]
https://github.com/mmanosalva/SublimeTeX done

**Reply by anamanuellar** [reply]
Done

**Reply by MuhammedM294** [reply]
https://github.com/MuhammedM294/Iteru Done!

**Comment by orzklv** [comment]
https://github.com/katsuki-yuri/neko done

**Reply by vinograd0v** [reply]
Done https://github.com/mmanosalva/SublimeTeX

**Reply by orzklv** [reply]
@mmanosalva done comrade!)

**Comment by MuhammedM294** [comment]
https://github.com/MuhammedM294/Iteru    Done!

**Reply by ArielMAJ** [reply]
@MuhammedM294 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by MuhammedM294** [reply]
@ArielMAJ Done!

**Reply by ArielMAJ** [reply]
Ty!

**Comment by ittid** [comment]
Done. Here's mine! https://github.com/ittid/ittiddev.com
